source/slang/slang-ir-glsl-legalize.cpp

```diff
@@ -265,7 +265,8 @@ struct GLSLLegalizationContext
     Stage                      stage;
     IRFunc*                    entryPointFunc;
     
-    //Dictionary<ResourceType, Dictionary<Space, BindingIndex>>
+    /// This dictionary stores for 'VaryingIn/VaryingOut' ('Dictionary<LayoutResourceKind, ...>') all used
```

Comment:
Why would a `varying in/out` has a `space` binding? They should only have a single location number, so it appears to me that `Dictionary<LayoutResourceKind, UIntSet>` is sufficient.

Reply:
> Why would a varying in/out has a space binding? 

They should not have a space other than 0, although, I did not see anywhere that the GLSL legalization code ensures any of this

Reply:
We can safely assume this, because there is no way in spirv/glsl to specify a "space" for a varying inout.

Reply:
I will add this change👍